include/seqan3/alignment/pairwise/detail/policy_optimum_tracker_simd.hpp

```diff
@@ -149,6 +156,37 @@ class policy_optimum_tracker_simd :
      * have different sizes. Thus, the end coordinate for each individual alignment matrix can differ. The score is
      * populated to the end of the matrix and only the respective coordinates are tracked for the optimal score.
      * Finally, the added offset is removed from the score to obtain the true value.
+     *
+     * ### Example
+     *
+     * Consider the following collections:
+     * collection 1: ["aaa", "aa",   "a"]
+     * collection 2: [  "a", "aa", "aaa"]
+     *
+     * Based on the length of the sequences the encompassing alignment matrix has the dimensions 4x4.
+     * The following graphic depicts this matrix. A `x` marks the last cell for the contained matrix given the
```

Comment:
There are no `x`'s right?

Reply:
it is supposed to be a wild card for the number.

Reply:
Maybe I call it the number, instead of A `x`